src/Components/CreatePlaylistModal/CreatePlaylistModal.js

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from 'react';
+import { usePlaylist } from '../../Context/PlaylistContext/PlaylistContext';
+import './CreatePlaylistModal.css'
+
+function CreatePlaylistModal({ showModal, closeModal}) {
+
+  const { createNewPlaylist } = usePlaylist()
+
+  const [Name, setName] = useState('')
```

Comment:
both states can be merged into a single state

Reply:
Sure will update it, thanks